.github/pull_request_template.md

```diff
@@ -0,0 +1,59 @@
+# Summary | Résumé
+
+> 1-3 sentence description of the changed you're proposing.
+> Please include the GitHub issue number or relevant acceptance criteria.
+> For any UI changes please include screenshots at different breakpoints.
+
+---
+
+> Description en 1 à 3 phrases de la modification proposée, avec un lien vers le
+> problème (« issue ») GitHub ou la fiche Trello, le cas échéant.
+
+# Test instructions | Instructions pour tester la modification
+
+> Sequential steps (1., 2., 3., ...) that describe how to test this change. This
+> will help a developer test things out without too much detective work. Also,
+> include any environmental setup steps that aren't in the normal README steps
+> and/or any time-based elements that this requires.
+
+---
+
+> Étapes consécutives (1., 2., 3., …) qui décrivent la façon de tester la
+> modification. Elles aideront les développeurs à faire des tests sans avoir à
+> jouer au détective. Veuillez aussi inclure toutes les étapes de configuration
+> de l’environnement qui ne font pas partie des étapes normales dans le fichier
+> README et tout élément temporel requis.
+
+# Help requested | Aide requise
+
+> Things that you (the submitter) want reviewers to pay very close attention to
+> when they review this. Please include as much context as possible
+
+---
+
+> Éléments auxquels vous (le demandeur) souhaitez que les réviseurs prêtent
+> attention lorsqu’ils examineront votre demande.
+
+# Unresolved questions / Out of scope | Questions non résolues ou hors sujet
+
+> Are there any related issues or tangent features you consider out of scope for
+> this issue that could be addressed in the future?
+
+---
+
+> Y a-t-il des questions connexes que vous considérez hors sujet et qui
+> pourraient être abordées plus tard?
+
+# Pull Request Checklist
+
+Please complete the following items in the checklist before you request a review:
+
+- [ ] Does it work? Is the PR solving the problem it's meant to solve?
```

Comment:
Maybe just a small language change around `Does it work?`... could be a little harsh.
How about 'Have you completely tested the functionality introduced with this PR' or something along those lines?